crates/viewer/re_renderer/shader/world_grid.wgsl

```diff
@@ -0,0 +1,154 @@
+#import <./global_bindings.wgsl>
+
+struct WorldGridUniformBuffer {
+    color: vec4f,
+
+    /// A value of [`super::GridPlane`]
+    orientation: u32,
+
+    /// How far apart the closest sets of lines are.
+    spacing: f32,
+
+    /// How thick the lines are in UI units.
+    thickness_ui: f32,
+}
+
+@group(1) @binding(0)
+var<uniform> config: WorldGridUniformBuffer;
+
+// See world_grid::GridPlane
+const ORIENTATION_XZ: u32 = 0;
+const ORIENTATION_YZ: u32 = 1;
+const ORIENTATION_XY: u32 = 2;
```

Comment:
extremely nitty nit but, as usual I can't help myself: why this order?

It would make more sense to me if the integer indicated the up-axis:

YZ = 0
ZX = 1
XY = 2

Reply:
I just went in like random so that's a good principle 😄